ush/forecast_postdet.sh

```diff
@@ -628,6 +628,36 @@ CICE_out() {
 
 }
 
+CICE_avg() {
+  if (( CICE_cdoavg == 0 )); then
+    echo "SUB ${FUNCNAME[0]}: No further averaging needed for CICE"
+  else
+    echo "SUB ${FUNCNAME[0]}: Averaging output data for CICE"
+    CDO=/apps/prod/hpc-stack/intel-19.1.3.304/cdo/1.9.8/bin/cdo
```

Comment:
If we choose to use the cdo utility for this, it may need to be added somewhere in `jobs/JGLOBAL_FORECAST`. This line is just a temporary workaround for the cdo utility being undefined in the forecast task environment.

Reply:
The cdo binary directory is already added to the path by the cdo module. You should be able to just call it with `cdo` without this.

Reply:
NB: The model modulefile isn't loading cdo, so it would need to be added to [load_ufswm_modules.sh](https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/load_ufswm_modules.sh).

Reply:
Thanks, @WalterKolczynski-NOAA. Will module load cdo in [load_ufswm_modules.sh](https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/load_ufswm_modules.sh).